frontend/src/components/App.js

```diff
@@ -10,14 +11,46 @@ import Signup from './Signup';
 import Navbar from './Navbar';
 import CreateApplicationPeriod from '../scenes/ApplicationPeriod/create';
 
+
 // eslint-disable-next-line react/prefer-stateless-function
 class App extends Component {
+  constructor(props, context) {
+    super(props, context);
+    this.state = {
+      loggedIn: false,
+      userToken: null,
+    };
+    this.cookies = new Cookies();
+    this.attemptLogOut = this.attemptLogOut.bind(this);
+    this.setLoginState = this.setLoginState.bind(this);
+  }
+
+  // Check cookie session here to keep logon whenever user reloads page
+  componentDidMount() {
+    if (this.cookies.get('userSession')) {
+      this.setState({ loggedIn: true });
+    }
+  }
+
+  setLoginState(token) {
+    const { userToken } = this.state;
+    console.log(userToken); // temporarily bypass elint, remove once used for retrieving user data from backend
```

Comment:
maybe put the comment on a different line, to appease the linter:

https://travis-ci.com/ubclaunchpad/pinpoint/jobs/180038402#L507

```suggestion
    // temporarily bypass elint, remove once used for retrieving user data from backend
    console.log(userToken);
```